client/src/pages/Landing/WhyUseUs/whyUseUs.module.scss

```diff
@@ -6,8 +6,8 @@
   gap: 30px;
   margin-top: 1.87rem;
   margin-bottom: 1.87rem;
-  @include px-to-rem(margin-top, 24);
-  @include px-to-rem(margin-bottom, 24);
+  @include px-to-rem(padding-top, 30);
+  @include px-to-rem(padding-bottom, 30);
```

Comment:
usually when the values are same, you can use short-hand

@include px-to-rem(padding, 30, 0);
will be
padding: 30 0;